var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -101,7 +101,7 @@ def patch(self):
 
     @property
     def common_config_args(self):
-        return [
+        config_args = [
             '-prefix', self.prefix,
```

Comment:
I believe, when I tried to use the common_config_args there were configure options which qt3 didn't support and the configure failed.   But that was a long time ago, so I'm not completely sure of my recollection.   I can try again.